### PR TITLE
Fix check args in CLI

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [x] document all added functions
* [x] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

### CODE REVIEW

File: massa-client/src/cmds.rs [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ impl Command {
                 }
             }
             Command::read_only_execute_smart_contract => {
-                if parameters.len() != 2 && parameters.len() <= 4 {
+                if parameters.len() < 2 || parameters.len() > 5 {
```
Comment (Leo-Besancon) [inline massa-client/src/cmds.rs:1116]: Unless I'm tired, it should be
```rust
if parameters.len() < 2 || parameters.len() > 4
```
right?
If you have 5 params it should bail ( read_only_execute_smart_contract PathToBytecode MaxGas Address IsFinal )

Comment (aoudiamoncef) [reply massa-client/src/cmds.rs:1116]: Copy and paste is evil when you don't have automated test. I'll recheck for the 100 time

Comment (aoudiamoncef) [reply massa-client/src/cmds.rs]: done

Comment (Leo-Besancon) [reply massa-client/src/cmds.rs]: Nice! Improving the client, one PR at a time 👍

**Review by Leo-Besancon — APPROVED** [review]

**Pull request merged**